src/bin/nydusd/main.rs

```diff
@@ -248,7 +248,15 @@ fn prepare_commandline_options() -> Command {
             Arg::new("log-rotation-size")
                 .long("log-rotation-size")
                 .help("Specify log rotation size(MB), 0 to disable")
-                .default_value("0")
+                .default_value("100")
+                .required(false)
+                .global(true),
+        )
+        .arg(
+            Arg::new("log-rotation-backups")
```

Comment:
At this stage, giving the ability to configure `log-rotation-size` at nydus-snapshotter side is acceptable with the hardcoded 10 backups

Reply:
If we use the hardcoded 10 backups, there no need to expose `log-rotation-backups` to users. I will close this PR later. 😄